test/src/data/amazon-blurb.js

```diff
@@ -0,0 +1,5 @@
+export default [
+    {
+        amazon_book_blurb: 'Support OpenStax with your purchase through Amazon.'
```

Comment:
just flagging that there is additional blurb text on [the card](https://github.com/openstax/openstax.org/issues/2365)